SweetsFactory/src/app/shared/services/user.service.ts

```diff
@@ -17,21 +17,34 @@ export class UserService {
         this.users = this.getUsers();
     }
 
-    userLogedin(user: User) {
+    userLoggedIn(user: User) {
+        this.currentUser = user;
         this.loginAnnouncedSource.next(user);
     }
 
     getUsers() {
         return USERS;
     }
 
+    isAdmin(): boolean {
+        return this.currentUser.isAdmin;
+    }
+
     isValidUser(userName: string, password: string): boolean {
+        this.isLogged = false;
         this.users.forEach(elem => {
             if (elem.name == userName && elem.password == password) {
```

Comment:
Please make a habit of using triple === when doing comparisons in JS - it's the recommended practice 👍